### ISSUE TITLE
current status and readme?

### ISSUE THREAD

**Issue by jianshu93** [imartayan/minisimd#1]
Dear @imartayan,

I am wondering what is the current status for this simd minimizer, and benchmark with the minimizer-queue? Is it ready for production purposes.

Thanks,
Jianshu

**Comment by imartayan** [comment]
Hello @jianshu93,
Thanks for reaching out!

This repo was a proof of concept to experiment with SIMD instructions to compute minimizers, I should probably archive it.
Since then, I made an improved version that was merged in Ragnar Groot Koerkamp's [minimizers](https://github.com/RagnarGrootKoerkamp/minimizers) library: https://github.com/RagnarGrootKoerkamp/minimizers/pull/4

Ragnar and I are working on a paper to present this SIMD approach in details.
I wouldn't say the library is production ready yet, but once the implementation is complete I plan to include it in a new release of [minimizer-iter](https://github.com/rust-seq/minimizer-iter).
In comparison to minimizer-queue, this version should be much faster (10-100x depending on your hardware).
